### PR TITLE
Ws example fix

### PR DESCRIPTION
**Pull request by fr0stbite**
## What do these changes do?

This minimal pull request fixes depreceated API usage in the `examples/web_ws.py` example.

## Are there changes in behavior for the user?

Users should now be able to run the `examples/web_ws.py` example without any problems.

## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] Add yourself to `CONTRIBUTORS.txt`
- [ ] Add a new entry to `CHANGES.rst`
  * Choose any open position to avoid merge conflicts with other PRs.
  * Add a link to the issue you are fixing (if any) using `#isuue_number` format at the end of changelog message. Use Pull Request number if there are no issues for PR or PR covers the issue only partially.

### CODE REVIEW

File: examples/web_ws.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import asyncio
 import os
 
-from aiohttp.web import (Application, Response, WebSocketResponse, WSMsgType,
+from aiohttp.web import (Application, MsgType, Response, WebSocketResponse,
```
Comment (asvetlov) [inline examples/web_ws.py]: `MsgType` is a deprecated name in favor of `WSMsgType`.

File: examples/web_ws.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
         request.app['sockets'].append(resp)
 
         async for msg in resp:
-            if msg.type == WSMsgType.TEXT:
+            if msg.type == MsgType.TEXT:
```
Comment (asvetlov) [inline examples/web_ws.py]: Replace with `WSMsgType` back

**Comment by asvetlov** [comment]
Thanks!

**Comment by fr0stbite** [comment]
Glad to be able to help!

**Pull request merged**